applications/wg-easy/charts/cert-manager/templates/secret-preflights.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cert-manager-preflights
+  labels:
+    troubleshoot.sh/kind: preflight
+type: Opaque
+stringData:
+  preflight.yaml: |
+    apiVersion: troubleshoot.sh/v1beta2
+    kind: Preflight
+    metadata:
+      name: cert-manager-preflights
+    spec:
+      analyzers:
+        # https://github.com/cert-manager/cert-manager/blob/master/deploy/charts/cert-manager/README.template.md#prerequisites
+        - clusterVersion:
+            outcomes:
+              - fail:
+                  when: "< 1.22.0"
+                  message: The application requires at least Kubernetes 1.22.0, and recommends 1.25.0.
+                  uri: https://kubernetes.io
```

Comment:
What is this uri reference? I don't feel like a generic reference to kubernetes.io is very helpful but I'm not clear what this is for.

Wouldn't a reference to the cert-manager docs bet more appropriate here?

Reply:
Thank you! I've updated the uri to https://cert-manager.io/docs/installation/helm